docs/ui/components/Header/Logo.tsx

```diff
@@ -2,37 +2,30 @@ import { css } from '@emotion/react';
 import {
   breakpoints,
   theme,
+  iconSize,
   spacing,
   typography,
+  ChevronRightIcon,
   Logo as LogoIcon,
   WordMarkLogo,
-  borderRadius,
 } from '@expo/styleguide';
 import React from 'react';
 
-import { H4, LinkBase } from '~/ui/components/Text';
+import { LinkBase } from '~/ui/components/Text';
 
 export const Logo = () => (
-  <div css={containerStyle}>
-    <LinkBase css={linkStyle} href="https://expo.dev">
+  <>
+    <LinkBase css={linkStyle} href="https://expo.dev" openInNewTab>
       <WordMarkLogo color={theme.text.default} css={[logoStyle, hideOnMobile]} />
       <LogoIcon color={theme.text.default} css={[logoStyle, showOnMobile]} />
     </LinkBase>
-    {/* <ChevronRightIcon size={iconSize.small} color={theme.icon.secondary} /> */}
+    <ChevronRightIcon size={iconSize.regular} css={chevronStyle} color={theme.icon.secondary} />
```

Comment:
```suggestion
    <ChevronRightIcon css={chevronStyle} color={theme.icon.secondary} />
```
nitpick: all the icons are this size by default

Reply:
Whoops, nice catch! This is due to revert of this code,, it was already addressed once. 😅